reference-implementation/to-upstream-wpts/writable-streams/bad-strategies.https.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script src="/streams/resources/test-initializer.js"></script>
+<style src="/resources/testharness.css"/>
+
+<script src="bad-strategies.js"></script>
+<script>
+'use strict';
+worker_test('bad-strategies.js');
+service_worker_test('bad-strategies.js');
```

Comment:
The "real" version is at https://github.com/w3c/web-platform-tests/blob/master/streams/resources/test-initializer.js